pkg/api/customization/project/project_actions.go

```diff
@@ -273,7 +273,7 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 
 	project, err := h.updateProjectPSPTID(request, podSecurityPolicyTemplateName)
 	if err != nil {
-		return fmt.Errorf("error updating PSPT ID: %v", err)
+		return httperror.WrapAPIError(err, httperror.Conflict, "error updating PSPT ID")
```

Comment:
you should only do this if the error returned from pudateProjectPSPTID was a conflict error.
```
if apierros.IsConflict(err) { ... }
```